vignettes/manageSynapseCredentials.Rmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "Manage Synapse Credentials"
+author: "KimYen Ladia"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Manage Synapse Credentials}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+## Manage Synapse Credentials
+
+There are multiple ways one can login to Synapse. We recommend users to choose the method that fits your workflow.
+
+### One Time Login
+
+Use username and password to login as follow:
+```{r collapse = TRUE, eval = F}
+library("synapser")
+synLogin("username", "password")
+```
+
+### Use `.synapseConfig`
+
+For writing code using the `synapser` client, that easy to share with others, please do not include your credentials in the code. Instead, please use `.synapseConfig` file. You can prefer this documentation to your collaborators on how to setup `.synapseConfig` file.
+
+First, reate a `.synapseConfig` file in your home directory with the following content:
+```{r collapse = TRUE, eval = F}
+[authentication]
+username = your_username
+password = your_password
+
+```
+
+Then login without specifying your username and password:
+```{r collapse = TRUE, eval = F}
+library("synapser")
+synLogin()
+```
+
+### Letting the Operating System Manage Your Synapse Credentials
+
+For users who would like to save their credentials and let other OS configured application (like keychain in Mac) manages credentials for you, when login the first time, use:
+```{r collapse = TRUE, eval = F}
+library("synapser")
+synLogin("username", "password", rememberMe=True)
+```
+
+Then the application (keychain in Mac) will prompt you to allow other applications including Python and R to access this credentials. Please choose "Yes" or "OK".
```

Comment:
Then the application (keychain in Mac)  -> The application (keychain in Mac)  will then prompt you
this credentials -> these credentials